App.svelte

```diff
@@ -78,6 +79,7 @@
       <DrawControls url={snapperUrl} />
       <HoverLayer />
       <EditingLayer />
+      <Legend />
```

Comment:
Was this missing previously? Was expecting a matching 

```
<Legend > 
```

Somewhere but maybe that's not how Svelte works...

Reply:
`Legend` is a totally new component, so just added here. `<x></x>` is equivalent to `<x />` in HTML generally, that's what's going on here

Reply:
Makes sense, `<Legend />` = `<Legend></Lengend>`, in HTML not Svelte? Learn something every day!